services/user_service/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8
+
+WORKDIR /user_service
```

Comment:
We won't access from any other place to your container, so it could be just /usr/app (or whatever you like)

Reply:
it makes sense